### PR TITLE
W-16723018 | FIPS release for AS2 and Rosettanet Connectors

### PR DESCRIPTION
**Pull request by RamaKrishna-4**
# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

File: as2/6.8/modules/ROOT/pages/as2-connector-upgrade-migrate.adoc [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+|===
+|From Version | To Version
+|5.x |6.x
+|===
+
+== Changes
+
+=== 6.8
+
+AS2 Connector 6.8 contains the following changes:
```
Comment (tasmoore) [inline as2/6.8/modules/ROOT/pages/as2-connector-upgrade-migrate.adoc]:
```suggestion
```

File: as2/6.8/modules/ROOT/pages/as2-connector-upgrade-migrate.adoc [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+
+* Additional validations on the keystore and algorithm usage to address reported security vulnerabilities.
```
Comment (tasmoore) [inline as2/6.8/modules/ROOT/pages/as2-connector-upgrade-migrate.adoc]:
```suggestion
Additional validations are now performed on the keystore and the algorithm usage to address reported security vulnerabilities.
```

File: as2/6.8/modules/ROOT/pages/as2-connector-upgrade-migrate.adoc [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+|From Version | To Version
+|5.x |6.x
+|===
+
+== Changes
+
+=== 6.8
+
+AS2 Connector 6.8 contains the following changes:
+
```
Comment (tasmoore) [inline as2/6.8/modules/ROOT/pages/as2-connector-upgrade-migrate.adoc]:
```suggestion
```

File: rosettanet/2.0/modules/ROOT/pages/index.adoc [hunk 4]
```diff
@@ -101,8 +101,7 @@ All configuration properties used by the RosettaNet connector are in the Studio
 * Production
 * Test
 * Unchecked
-| Object store reference |Optional reference to an object store definition to be used for storing messages awaiting acknowledgment. If not set, the connector always uses the default persistent object store to retain sent messages waiting for acknowledgments or retries. If set, the referenced bean must be an object store configuration to be used instead. See also:
-https://forums.mulesoft.com/questions/38011/what-is-an-object-store-bean.html[What is an object store bean?]
+| Object store reference |Optional reference to an object store definition to be used for storing messages awaiting acknowledgment. If not set, the connector always uses the default persistent object store to retain sent messages waiting for acknowledgments or retries. If set, the referenced object store configuration will be used.
```
Comment (tasmoore) [inline rosettanet/2.0/modules/ROOT/pages/index.adoc:104]:
```suggestion
| Object store reference |Optional reference to an object store definition to be used for storing messages awaiting acknowledgment. If not set, the connector always uses the default persistent object store to retain sent messages waiting for acknowledgments or retries. If set, the connector uses the referenced object store configuration instead.
```

File: rosettanet/2.1/modules/ROOT/pages/index.adoc [hunk 5]
```diff
@@ -0,0 +1,212 @@
...
+| Keystore path |Absolute file path or classpath relative to the project `/src/main/resources` directory for the keystore used for validating and/or signing messages (required when sending or receiving signed messages).
+| Keystore access password |Password used to protect the keystore (required when sending or receiving signed messages).
+| Signing key password |Password used to access the private signing key within the keystore (required when sending signed messages).
+| Force message signing |Optional override to force the use of signed messages even when not required by the PIP definition. Leave this value empty to the PIP definition for signing messages, set to `NEVER` to never sign any messages, or set to  `ALWAYS` to always sign messages.
+| Global usage code |Mode to run the connector, one of:
+
+* Production
+* Test
+* Unchecked
+| Object store reference |Optional reference to an object store definition to be used for storing messages awaiting acknowledgment. If not set, the connector always uses the default persistent object store to retain sent messages waiting for acknowledgments or retries. If set, the referenced object store configuration will be used.
```
Comment (tasmoore) [inline rosettanet/2.1/modules/ROOT/pages/index.adoc]:
```suggestion
| Object store reference |Optional reference to an object store definition to be used for storing messages awaiting acknowledgment. If not set, the connector always uses the default persistent object store to retain sent messages waiting for acknowledgments or retries. If set, the connector uses the referenced object store configuration instead.
```

File: rosettanet/2.0/modules/ROOT/pages/index.adoc (hunk 4 above)
Comment (tasmoore) [inline rosettanet/2.0/modules/ROOT/pages/index.adoc:103]: Should this change be reverted since it was made on the 2.0 version?

Comment (RamaKrishna-4) [reply rosettanet/2.0/modules/ROOT/pages/index.adoc:103]: No this link is broken and points to an article in Mule3 so this is incorrect in v2.0 as well.

File: rosettanet/2.1/modules/ROOT/pages/index.adoc [hunk 6]
```diff
@@ -0,0 +1,212 @@
...
+To use the RosettaNet connector in a production environment, you must
+have purchased a MuleSoft license for Anypoint B2B.
+
+== What's New in this Connector
+
+RosettaNet connector 2.x for Mule 4 makes several changes to the configuration and handling over what was implemented by RosettaNet connector 1.x:
+
+* Metadata supplied automatically for sources (no need for the separate metadata definition component used in 1.x)
+* Simplified flow structure, with separate flows used for each partner and message.
+* Additional validations on the keystore usage to address reported security vulnerabilities.
```
Comment (tasmoore) [inline rosettanet/2.1/modules/ROOT/pages/index.adoc]:
```suggestion
* Additional validations are now performed on the keystore usage to address reported security vulnerabilities. 
```

File: as2/6.8/modules/ROOT/pages/as2-connector-upgrade-migrate.adoc [hunk 7]
```diff
@@ -0,0 +1,149 @@
...
+|===
+|From Version | To Version
+|5.x |6.x
+|===
+
+== Changes
+
+=== 6.8
+
+Additional validations are now performed on the keystore and the algorithm usage to address reported security vulnerabilities.
```
Comment (tasmoore) [inline as2/6.8/modules/ROOT/pages/as2-connector-upgrade-migrate.adoc]:
```suggestion
Additional validations are now performed on the keystore and algorithm usage to address reported security vulnerabilities.
```

Comment (tasmoore) [reply as2/6.8/modules/ROOT/pages/as2-connector-upgrade-migrate.adoc]: Minor change after viewing this sentence in the AS2 release notes

**Review by bjoson-mulesoft — APPROVED** [review]

**Review by bjoson-mulesoft — DISMISSED** [review]

**Review by Crispy-Salesforce — APPROVED** [review]

**Pull request merged**